readme.md

```diff
@@ -174,10 +190,41 @@ console.log(chalk.styles.red.open + 'Hello' + chalk.styles.red.close);
 - `bgWhite`
 
 
-## 256-colors
+## 256/16 million (Truecolor) color support
+
+Chalk supports 256 colors and, when manually specified, [Truecolor (16 million colors)](https://gist.github.com/XVilka/8346728) on all supported terminal emulators.
+
+For the methods that support it (listed below), the color will be 'fit' to the color level supported (i.e. RGB colors will be downsampled to 16 colors if only basic support is enabled).
+
+Some examples:
+
+For a complete list of color models, see [`color-convert`'s list of conversions](https://github.com/Qix-/color-convert/blob/master/conversions.js). Background versions of these models are prefixed with `bg` and the first level of the module capitalized (e.g. `keyword` for foreground colors and `bgKeyword` for background colors).
+
+- `chalk.hex('#DEADED').underline('Hello, world!')`
+- `chalk.keyword('orange')('Some orange text')`
+- `chalk.rgb(15, 100, 204).inverse('Hello!')`
+
+
+- `chalk.bgHex('#DEADED').underline('Hello, world!')`
+- `chalk.bgKeyword('orange')('Some orange text')`
+- `chalk.bgRgb(15, 100, 204).inverse('Hello!')`
 
-Chalk does not support anything other than the base eight colors, which guarantees it will work on all terminals and systems. Some terminals, specifically `xterm` compliant ones, will support the full range of 8-bit colors. For this the lower level [ansi-256-colors](https://github.com/jbnicolai/ansi-256-colors) package can be used.
+As of this writing, these are the supported color models that are exposed in Chalk:
 
+- `rgb`
+- `hex`
+- `keyword` (CSS keywords)
+- `hsl`
+- `hsv`
+- `hwb`
+- `cmyk`
+- `xyz`
+- `lab`
+- `lch`
+- `ansi16`
+- `ansi256`
+- `hcg`
+- `apple` (see [qix-/color-convert#30](https://github.com/Qix-/color-convert/issues/30))
```

Comment:
The other thing is that the way color-convert is versioned all new models that are added 'just work' with whatever is using them assuming you're using `^`. This would be an ever changing list.